lib/sentry/plug_context.ex

```diff
@@ -211,8 +211,15 @@ defmodule Sentry.PlugContext do
 
   defp remote_port(conn) do
     case Plug.Conn.get_req_header(conn, "x-forwarded-for") do
-      [] -> nil
-      [_value | _rest] -> Plug.Conn.get_peer_data(conn).port
+      [] ->
+        nil
+
+      [_value | _rest] ->
+        try do
```

Comment:
Can we leave a comment here saying that Bandit raises in some cases here and maybe point to the source code like you did in the PR description?

Reply:
thanks for the comment @whatyouhide , I've updated the commit.